examples/js/meta-touch-controls.js

```diff
@@ -0,0 +1,507 @@
+/* global AFRAME, THREE */
```

Comment:
note: I added this file only so that I could test with it on glitch: https://naf-wip.glitch.me/tracked-controllers.html

it is removed in a later commit